test/functional/rpc_fundrawtransaction.py

```diff
@@ -695,6 +739,7 @@ def test_address_reuse(self):
         ################################
         # Test no address reuse occurs #
         ################################
+        self.log.info("Testing fundrawtxn no address reuse occurs")
```

Comment:
this sentence sounds a bit clunky. Perhaps "Test no address reuse occurs with `fundrawransaction`"

Reply:
Agree, replacing with "Test fundrawtxn does not reuse addresses" to maintain consistent verb + op noun logging sentences.